cmd/fleetctl/apple_mdmd_test.go

```diff
@@ -0,0 +1,102 @@
+package main
```

Comment:
Is the last `d` in `apple_mdmd` a typo? Not sure what it means.

Reply:
lol thank you! it's a typo 😅